factory/status/statusComponents_test.go

```diff
@@ -68,6 +68,20 @@ func createMockStatusComponentsFactoryArgs() statusComp.StatusComponentsFactoryA
 	}
 }
 
+func createStatusFactoryArgs() statusComp.StatusComponentsFactoryArgs {
+	cfg := testscommon.GetGeneralConfig()
```

Comment:
A lot of this setup code is duplicated everywhere.
Like this is mostly the same code as in `createStateFactoryArgs`.

This applies globally for all these newly funcs !